lib/server.js

```diff
@@ -1538,6 +1539,7 @@ WalletService.prototype.getTxHistory = function(opts, cb) {
         fees: tx.fees,
         time: tx.time,
         addressTo: addressTo,
+        outputs: _.filter(outputs, { isChange: false }),
```

Comment:
the decorations `isMine` and `isChange` are returned with the outputs by bws. what do you think?